test/extended/util/annotate/generated/zz_generated.annotations.go

```diff
@@ -47,8 +47,12 @@ var Annotations = map[string]string{
 
 	"[sig-api-machinery][Feature:ClusterResourceQuota] Cluster resource quota should control resource limits across namespaces [apigroup:quota.openshift.io][apigroup:image.openshift.io][apigroup:monitoring.coreos.com][apigroup:template.openshift.io]": " [Suite:openshift/conformance/parallel]",
 
+	"[sig-api-machinery][Feature:ResourceQuota] Object count check the quota after import-image with --all option [Skipped:Disconnected]": " [Suite:openshift/conformance/parallel]",
+
 	"[sig-api-machinery][Feature:ResourceQuota] Object count should properly count the number of imagestreams resources [apigroup:image.openshift.io]": " [Suite:openshift/conformance/parallel]",
 
+	"[sig-api-machinery][Feature:ResourceQuota] Object count should properly count the number of persistentvolumeclaims resources [Serial]": " [Suite:openshift/conformance/serial]",
```

Comment:
Note: this is not part of backported commit dde58284a7c5009403c30c6a7b328a575b00b8d9
Can you tell is this is intentional @gangwgr ?

Reply:
@flavio-fernandes Earlier it was backported but it get reverted back when this pr is reverted back https://github.com/openshift/origin/pull/2830